tests/test_wog.py

```diff
@@ -180,6 +180,18 @@ def test_cult_apothecary():
 	assert game.player1.hero.health == 28
 
 
+def test_deathwing_dragonlord():
+	game = prepare_empty_game()
+	deathwing = game.player1.give("OG_317")
+	game.player1.give("EX1_572")
+	game.player1.give("EX1_284")
+	game.player1.give(WISP)
+	deathwing.play()
+	deathwing.destroy()
+	assert len(game.player1.hand) == 1
+	assert len(game.player1.field) == 2
```

Comment:
Stuff those cards in variables and compare the hand and field to lists, eg: `game.player1.hand == [one, two]`.

Incidentally this will check the ordering of the minions on the field. Cross your fingers we're getting that right :)